apps/examples/testcase/le_tc/filesystem/tc_fs_procfs.c

```diff
@@ -33,16 +33,20 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include "tc_common.h"
+#ifndef CONFIG_FS_AUTOMOUNT_PROCFS
+#include <stdbool.h>
+#endif
 
-#define PROC_MOUNTPOINT "/proc"
+#define PROCFS_TEST_MOUNTPOINT "/proc"
```

Comment:
How about "/test_proc" or "/proc_test"? "/proc" is a too common name so that it is easily duplicated with user's proc mount point.